### PR TITLE
Add Baseline (compatibility) glossary entry

### PR DESCRIPTION
**Pull request by ddbeck**
### Description

This adds a glossary entry for the Baseline term (and, soon, iconography) for forthcoming web platform feature compatibility status summaries.

cc: @Rumyra, @atopal, @LeoMcA since they've previewed much of this and have additional context

### Motivation

In the not-too-distant future, some MDN pages will have Baseline status information. These statuses are not (yet) widely recognized or self-explanatory; this glossary entry will provided a reader-facing description of the status summary, its origins, and how a reader might affect its contents.

### Related issues and pull requests

- https://github.com/mdn/yari/pull/8772
- https://github.com/web-platform-dx/feature-set#readme

### CODE REVIEW

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+A Baseline feature—such as an API, a set of CSS properties, or a JavaScript syntax—is known to work consistently in the current and previous major versions of several well-known browsers (Chrome, Edge, Firefox, and Safari).
+
+## Badges
+
+<!-- TODO: Show Baseline indicator itself, once it has been merged -->
+🟢 If you see the green Baseline badge, then you can trust that the feature will work in the browsers' latest and previous major releases.
+
+<!-- TODO: Show the non-Baseline indicator itself, once it has been merged -->
+🟡 If you see a yellow badge showing that a feature is not yet Baseline, then do more research and testing with your site's users before relying on that feature, or wait for it to become Baseline.
```
Comment (ddbeck) [inline files/en-us/glossary/baseline/compatibility/index.md:17]: I'm happy to update this after the fact, but there's no (merged) assets to use yet.

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Baseline (compatibility)
+slug: Glossary/baseline/compatibility
+---
+
+**Baseline** identifies widely-supported web platform features.
+Web developers can expect Baseline features to work in contemporary browsers.
+
+A Baseline feature—such as an API, a set of CSS properties, or a JavaScript syntax—is known to work consistently in the current and previous major versions of several well-known browsers (Chrome, Edge, Firefox, and Safari).
```
Comment (Rumyra) [inline files/en-us/glossary/baseline/compatibility/index.md:9]:
```suggestion
A Baseline feature - such as an API, a set of CSS properties, or a JavaScript syntax - is known to work consistently in the current and previous major versions of several well-known browsers (Chrome, Edge, Firefox, and Safari).
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: files/en-us/glossary/baseline/compatibility/index.md (hunk 2 above)
Comment (Rumyra) [reply files/en-us/glossary/baseline/compatibility/index.md:9]: Not sure if this was intended, but assumed not 👍

**Review by Rumyra — APPROVED** [review]

File: files/en-us/glossary/baseline/index.md [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 
-The descenders of characters like g and p extend below this line. {{Glossary("glyph", "Glyphs")}} with rounded lower and upper extents like C or 3 slightly extend below it.
-
-East Asian scripts have no baseline. Their glyphs are placed in a square box without ascenders or descenders.
+{{GlossaryDisambiguation}}
 
 ## See also
 
-- [Baseline](<https://en.wikipedia.org/wiki/Baseline_(typography)>) on Wikipedia
-- [CSS Box Alignment](/en-US/docs/Web/CSS/CSS_Box_Alignment#types_of_alignment) on MDN
```
Comment (estelle) [inline files/en-us/glossary/baseline/index.md:16]: It is a value in box alignment and.  See Glossary/Fallback_Alignment 
See also CanvasRenderingContext2D.textBaseline. 
It's also in vertical align in tables and (should likely not be mentioned)

Comment (ddbeck) [reply files/en-us/glossary/baseline/index.md:16]: @estelle these changes are an artifact of moving the contents of the entry and creating a disambiguation page. The Glossary/Baseline/Typography entry still contains the link to CSS Box Alignment

**Pull request merged**